### ISSUE TITLE
Create UE4 Project

### ISSUE THREAD

**Issue by maishiroma** [maishiroma/Project_Brng#4]

**Comment by maishiroma** [comment]
Oh yikes, just ran into a problem that could have gone really south. So after I decided to remove the starter content from the project, since most of them are 3D items, I got an email from GittHub saying that I went over my Git LFS limit for storage. Upon checking it out, I realized that my last transaction caused a surplus of Git LFS to be stored.

Thankfully, I already purged that content + all of those items from the repo __before__ I merged my branch, but it seemed that GitHub still has that tracked somehow? I checked the repo again using `git-lfs ls-files` and I only saw a the last files I put up (which are really small considering). So my guess is that the alert is for the initial transaction; the actual files should be purged from GitHub within the next 3 days.

TL;DR, be careful with Unreal Engine and large files. If anything, I guess this really does show that UE4 does not play nice with GitHub.
